src/yamlfix/services.py

```diff
@@ -18,18 +18,29 @@
 Files = Union[Tuple[TextIOWrapper], List[str]]
 
 
+class RuamelOptions(object):
```

Comment:
Maybe this file belongs better in a `model.py` file. I hadn't created it as so far we didn't have any models